src/main/scala/io/iohk/ethereum/db/components/StoragesComponent.scala

```diff
@@ -19,7 +19,7 @@ trait StoragesComponent {
 
     val mptNodeStorage: MptNodeStorage
 
-    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage but
+    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage, should we keep only one? [EC-147]
```

Comment:
I agree, there's already a task created for looking at this [EC-147](https://iohk.myjetbrains.com/youtrack/issue/EC-147) :+1: